### DISCUSSION TITLE
Le format de la date n'est pas bien localisé sur mycompanyinfrance

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2678] [Idées]
![image](https://user-images.githubusercontent.com/1775934/136181361-338d5f90-84a5-42b1-a7df-f9fa28a87426.png)

**Comment by lorenzoalali** [comment]
La page de stats EN a aussi un mix de FR et EN

![image](https://user-images.githubusercontent.com/11087992/136195950-a7c811b9-e7a6-4c35-92bd-483d9c3de4af.png)
